addons/mail/static/src/discuss/call/common/call_permission_dialog.scss

```diff
@@ -0,0 +1,46 @@
+.o_dialog .o_mail_discuss_permissions_image.o_animate {
```

Comment:
- `.o_dialog` should not be necessary
- The class name should match the component name (that should match the file name). This ensures that the code is readable and behaves "predictably"